modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/StorageEngine.java

```diff
@@ -43,6 +43,11 @@ public interface StorageEngine {
      */
     void stop() throws StorageException;
 
+    /**
+     * Whether the data is lost upon engine restart or not.
+     */
```

Comment:
```suggestion
    /** Whether the data is lost upon engine restart or not. */
```

Reply:
Every other method in this interface has multi-lined javadoc. What's your motivation here?

Reply:
I didn’t know that you could use a one-line JavaDoc for methods and classes, I think it’s more beautiful and reduces the lines of code in the file.

Reply:
Reducing the lines is not the goal. Usually it's quite the opposite, trying to cram everything into a smaller number of lines makes code significantly worse.
Beauty is subjective. I like when code looks homogeneous, these's a sense of order and structure in that case. When every method looks different, it just feels wrong to me. That's my part of the motivation.

Reply:
Everyone will have their own opinion.